modules/users/client/controllers/settings/change-password.client.controller.js

```diff
@@ -33,11 +33,13 @@
       // If successful show success message and clear form
       $scope.$broadcast('show-errors-reset', 'vm.passwordForm');
       vm.success = true;
```

Comment:
@mleanos @simison I added about 3 tests on user e2e that is checking for the notification and it's contents.

But I am stuck on these client controller tests. If we remove the vm.error from the controller and intend to continue unit testing the controllers for error we would need to check for the notification. The notification is DOM element which I don't have access to without using jasmine jquery. I know we can access DOM in protractor  e2e tests. But any guidance on how I can check for the notification on the controller tests?